File: ide/syntax/syntaxtest/testdata_rust_module/src/geometry.rs
pub const PI: f64 = 3.141592653589793;

pub fn area(radius: f64) -> f64 {
    PI * radius * radius
}

// Defined but never referenced as geometry::circumference, so it resolves
// only through the definitions phase.
pub fn circumference(radius: f64) -> f64 {
    2.0 * PI * radius
}

pub struct Point {
    pub x: f64,
    pub y: f64,
}

File: internal/ide/idelsp/symbolresolve/testdata_py/geometry.py
import math


def area(radius):
    return math.pi * radius * radius


def perimeter(radius):
    # Defined but never referenced as geometry.perimeter; resolves
    # only through the definitions phase.
    return 2 * math.pi * radius


class Shape:
    def __init__(self, size):
        self.size = size

    def area(self):
        return self.size * self.size


class _Internal:
    # Leading-underscore name: Python has no enforced visibility, so
    # the resolver must still surface it.
    pass


def _make_internal():
    return _Internal()
